packages/admin-ui/ui/src/domain/orders/details/detail-cards/summary.tsx

```diff
@@ -28,9 +34,49 @@ const SummaryCard: React.FC<SummaryCardProps> = ({ order, reservations }) => {
   } = useToggleState()
 
   const { showModal } = useContext(OrderEditContext)
+  const { client } = useMedusa()
   const { isFeatureEnabled } = useFeatureFlag()
   const inventoryEnabled = isFeatureEnabled("inventoryService")
 
+  const [variantInventoryMap, setVariantInventoryMap] = React.useState<
+    Map<string, VariantInventory>
+  >(new Map())
+
+  React.useEffect(() => {
+    if (!inventoryEnabled) {
+      return
+    }
+
+    const fetchInventory = async () => {
+      const inventory = await Promise.all(
+        order.items.map(async (item) => {
+          if (!item.variant_id) {
+            return
+          }
+          return await client.admin.variants.getInventory(item.variant_id)
```

Comment:
question: I am not sure if we've already talked about this, so I will ask again 😅
 
Is there a reason for not having a hook?

Reply:
We do have a hook for it, it's just because it's used in a method at this instance and hooks can't be used in method. Just the rules of react.

I think we could add an endpoint to list inventory for a list of variant id's, I just found this to be a faster solution than implementing an endpoint and all the hooks given the deadline is creeping up.

Reply:
> I think we could add an endpoint to list inventory for a list of variant id's

That sounds like a great idea :)

Reply:
putting it in the backlog in mw for now though 😄